calculator.html

```diff
@@ -0,0 +1,2178 @@
+       * 1000/4096 ~= 0.2442 and 100/4096 ~= 0.02442
+       * use 0.02442 to approximate "Top 100" (green) when there are less than 4096 results
+       * use 0.2442 to approximate "Top 1000" (orange) when there are less than 4096 results */
+    #outTable tr.good>*,
+    #familyTable tr.good>*,
+    #tradeTable tr.highMaxRow>* {
+      background-color: var(--goodResult);
+      color: var(--navActiveText);
+    }
+
+    #outTable tr.ok>*,
+    #familyTable tr.ok>*,
+    #tradeTable tr.okMaxRow>* {
+      background-color: var(--okResult);
+      color: var(--navActiveText);
+    }
+
+    #outTable tr.rubbish>*,
+    #familyTable tr.rubbish>*,
+    #tradeTable tr.lowMaxRow>* {
+      background-color: var(--badResult);
+      color: var(--navActiveText);
+    }
+
+    /* the > * operator applies the css to all children (cells) */
+    #outTable,
+    #familyTable a {
+      color: var(--navActiveText);
+    }
+
+    input[type=number] {
+      width: 44px;
+    }
+
+    input[id=pokeList] {
+      width: 111px;
+    }
+
+    /* Plus/Minus Buttons */
+    /* https://stackoverflow.com/q/33766350 */
+    .circle {
+      border: 1px solid var(--navBackground);
+      box-shadow: inset 1px 1px 1px var(--tableDividers);
+      width: 18px;
+      height: 18px;
+      border-radius: 100%;
+      position: relative;
+      margin: 4px;
+      display: inline-block;
+      vertical-align: middle;
+      background: #aaaaaa4f;
+    }
+
+    .circle:hover {
+      background: radial-gradient(var(--textBoxBg), var(--tableHeaderBg));
+    }
+
+    .circle:active {
+      background: var(--navActive);
+    }
+
+    .circle:before,
+    .circle:after {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    }
+
+    /* PLUS */
+    .circle.plus:before,
+    .circle.plus:after {
+      background: green;
+    }
+
+    .circle.plus:before {
+      width: 2px;
+      margin: 3px auto;
+    }
+
+    .circle.plus:after {
+      margin: auto 3px;
+      height: 2px;
+    }
+
+    /* MINUS */
+    .circle.minus:before {
+      background: #cc0000;
+      margin: auto 3px;
+      height: 2px;
+    }
+
+    #totalRanks {
+      display: inline;
+    }
+
+    p[id=IVinputShow],
+    small[id=IVinputHide] {
+      text-decoration: underline;
+      text-decoration-color: blue;
+    }
+  </style>
+  <!-- Begin Playwire Head -->
+  <!-- <script>window.ramp=window.ramp ||{}; window.ramp.que=window.ramp.que || []; </script> <script async src="//cdn.intergient.com/1025030/74482/ramp_config.js"></script> <script>window._pwGA4PageviewId=''.concat(Date.now()); window.dataLayer=window.dataLayer || []; window.gtag=window.gtag || function (){dataLayer.push(arguments);}; gtag('js', new Date()); gtag('config', 'G-JD7R9RWFZL',{'send_page_view': false}); gtag( 'event', 'ramp_js',{'send_to': 'G-JD7R9RWFZL', 'pageview_id': window._pwGA4PageviewId}); </script> -->
```

Comment:
Commented this out because of local errors but probably need to uncomment

Reply:
Yup this is the ad company's tags so they can track clicks / pay out for views on their ads